lokalise-file-push/action.yml

```
@@ -23,6 +23,8 @@ inputs:
     description: Pull request description
   keys-description:
     description: Description for keys in Lokalise
+  stale-unused:
+    description: Should be true if you want to stale unused keys from main branch
```

I'm not sure what this sentence means. Does it mean:

```suggestion
    description: Set to true if you want to mark unused keys in Lokalise as stale when merging with the main branch
```

Is it to the project itself to configure when this happens? If so maybe we could not mention the branch name

```suggestion
    description: Set to true if you want to mark unused keys in Lokalise as stale
```